solutions/core-landing-zone/setters.yaml

```diff
@@ -106,5 +106,20 @@ data:
   retention-in-days: "1"
   #
   ##########################
+  # DNS
+  ##########################
+  #
+  dns-project-id: dns-project-12345
+  dns-name: "dns-name.example.com."
```

Comment:
minor: the value could be just `example.com.`...... the client zone could then be : `client-name.example.com`

Reply:
Fixed. Please check if it's good.

Reply:
Modified as suggested, good one!